src/stars.css

```diff
@@ -0,0 +1,34 @@
+.card__star-container {
+  margin-top: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 16px;
+  font-weight: 400;
+  font-size: 10px;
+}
```

Comment:
`.card__star-container` by the name - the part of card block, please move this code to `card.css`.